src/webviews/webview-side/ipywidgets/common/kernel.ts

```diff
@@ -393,9 +390,6 @@ class ProxyKernel implements IMessageHandler, Kernel.IKernelConnection {
     }
     private messageHookInterceptor(msg: KernelMessage.IIOPubMessage): boolean | PromiseLike<boolean> {
         try {
-            logMessageOnlyOnCI(
```

Comment:
Figured if we needed these later, we can add a new `IPYWidgets_LogMessageIfCI` message. Then on the other side  it would only log these if running on CI.